purpleair-aqi.js

```diff
@@ -49,6 +48,44 @@ const DEFAULT_SENSOR_ID = 69223;
  * @property {number} longitude
  */
 
+/**
+ * Get JSON from a local file
+ *
+ * @param {string} fileName
```

Comment:
@a-ogilvie I tried my best to add structure comments, but, I definitely don't use them.

Reply:
Looks good to me! 🎉